requirements.txt

```diff
@@ -1 +1,2 @@
-hazelcast-python-client~=4.0.0
\ No newline at end of file
+hazelcast-python-client~=4.0.0
```

Comment:
Only the second line should be enough, we can delete `hazelcast-python-client~=4.0.0`

Reply:
Done, thank you 👍